src/Command/PqCommands/ContentHubPqSingleTranslation.php

```diff
@@ -0,0 +1,187 @@
+      $entities = $this->loadEntities($etm, $entityType);
+      $singleTranslation = [];
+      foreach ($entities as $entity) {
+        if ($entity->isTranslatable() && count($entity->getTranslationLanguages()) === 1) {
+          $singleTranslation[$entity->bundle()]['count']++;
+          $singleTranslation[$entity->bundle()]['langcode'] = $entity->language()->getId();
+
+        }
+      }
+
+      $formatted = [];
+      foreach ($singleTranslation as $bundleId => $bundle) {
+        $formatString = $this->toYellow('%s: %s: %s');
+        $formatted[] = sprintf($formatString, $bundles[$bundleId]['label'], $bundle['langcode'], $bundle['count']);
+      }
+
+      $kriMessage = !empty($formatted) ? PqCommandResultViolations::$singleTranslation : 'No single-translation entities.';
+      $result->setIndicator(
+        $kriName,
+        implode("\n", $formatted),
+        $kriMessage,
+        !empty($formatted)
+      );
+    }
+    return 0;
+  }
+
+  /**
+   * Returns bundles for entity type.
+   *
+   * @param string $entityType
+   *   Entity type id.
+   * @param \Drupal\Core\Entity\EntityTypeBundleInfoInterface $bundleInfo
+   *   Bundle information.
+   *
+   * @return array
+   *   The list of bundles.
+   *
+   * @throws \Exception
+   */
+  public function getBundles(string $entityType, EntityTypeBundleInfoInterface $bundleInfo): array {
+    $bundleIds = [];
+    $bundles = $bundleInfo->getBundleInfo($entityType);
+    foreach ($bundles as $bundleId => $bundle) {
+      $bundleIds[$bundleId]['label'] = $bundle['label'];
+      $bundleIds[$bundleId]['translatable'] = $bundle['translatable'];
+    }
+    return $bundleIds;
+  }
+
+  /**
+   * Returns entity type label.
+   *
+   * @param string $entityType
+   *   Entity type id.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   Entity type manager.
+   *
+   * @return string
+   *   Entity type label.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  protected function getEntityTypeLabel(string $entityType, EntityTypeManagerInterface $entityTypeManager): string {
+    $entityDefinition = $entityTypeManager->getDefinition($entityType);
+    return !$entityDefinition ? '' : $entityDefinition->getLabel()->__toString();
+  }
+
+  /**
+   * Loads all the entities and returns them.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $etm
+   *   Entity type manager.
+   * @param string $entityType
+   *   The entity type.
+   *
+   * @return \Drupal\Core\Entity\ContentEntityInterface[]
+   *   Entities of given entity type.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  protected function loadEntities(EntityTypeManagerInterface $etm, string $entityType): array {
+    $entity_storage = $etm->getStorage($entityType);
+    if (is_null($entity_storage)) {
+      return [];
+    }
```

Comment:
This is a bit redundant here. You already check if the entity type is valid before the process can get to the `loadEntities`. 
Also based on the documentation and the implementation this will not return NULL, but and exception.